docs/manual/manual_angle_control_mechanisms.rst

```diff
@@ -0,0 +1,647 @@
+    :align: center
+    :scale: 110%
+    :alt: Linear Displacement with a Gear Box
+
+    Linear Displacement with a Gear Box
+
+Meters Per Radian Demonstration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+First, if you have not already, please set up your computer to use Vertiq's Python API with the instructions found :ref:`here <getting_started_python_api>`.
+
+#. Gearbox Configuration with Meters Per Radian
+    In this example, we will demonstrate the effect of setting meter_per_rad and using linear displacements to control a gearbox output.
+    We will set meter_per_rad to 1/5 meaning that for every module radian traveled, we expect 1/5 radians of output (a 5:1 gear ratio). 
+    Running the sample below, we expect that the printed Angular Displacement is 5 times the Linear Displacement. 
+    By commanding linear displacement to :math:`\frac{\pi}{5}` we expect an observed angular displacement of :math:`\pi`.
+
+    .. note:: 
+        You must set the ``serial_port`` parameter to the serial port connected to your module.
+
+    .. code-block:: python
+        :linenos:
+
+        import iqmotion as iq
+        import numpy as np
+        import time
+
+
+        #Create a SerialCommunicator object for our servo module
+        com = iq.SerialCommunicator("COM3")
+        module = iq.ServoModule(com)
+
+
+        #Set "meter_per_rad" to our gear ratio
+        module.set("multi_turn_angle_control", "meter_per_rad", 1 / 5)
+
+
+        #For safety, set our maximum angular speed to 50 rad/s
+        module.set("multi_turn_angle_control", "angular_speed_max", 50)
+
+
+        #Make sure we're starting from 0 and give it time to finish
+        module.set("multi_turn_angle_control", "ctrl_angle", 0)
+        time.sleep(1)
+
+
+        #Tell the linear output to go to pi / 5 radians
+        module.set("multi_turn_angle_control", "ctrl_linear_displacement", np.pi / 5)
+        time.sleep(1)
+
+
+        #Print out the Angular Displacement and Linear Displacement
+        print("Angular Displacement: " , module.get("multi_turn_angle_control", "obs_angular_displacement"))
+        print("Linear Displacement: " , module.get("multi_turn_angle_control", "obs_linear_displacement"))
+
+    | The actual printed values are exactly what we expect:
+    | Angular Displacement:  3.14
+    | Linear Displacement:  0.63
+
+Maximum Angular Speed
+++++++++++++++++++++++++++++
+Vertiq servo modules define a value called *Angular Max Speed* :ref:`in the Multi Turn Angle Control Client <multi_turn_table>` which is the maximum travel speed permitted during all commanded module motion. 
+All angle commands are bound by this value, and **will not** exceed it. *Angular Max Speed* is the angular speed at which the module will 
+attempt to complete Control Angle as well as Control Linear Displacement commands. For example, if *Angular Max Speed* is set to 50 rad/s, 
+then a Control Angle of 25 rad will take about 0.5 seconds. 
```

Comment:
Probably obvious, but a control angle of 25 radians will take 0.5 seconds assuming you start from 0 displacement. May be good to specify